src/main/java/seedu/expense/model/expense/Amount.java

```diff
@@ -16,11 +16,12 @@
             "Expense Amount restrictions:\n"
             + "Amount Format: should only contain numbers and 1 '.', and should be in the <dollars>.<cents> format.\n"
             + "    \".<cents>\" input is optional but <dollars> should contain at least 1 digit.\n"
-            + "Value Restrictions: 0 to 10e9 inclusive";
-    public static final String VALIDATION_REGEX = "(?<dollars>\\d+)(.(?<cents>\\d{1,2}))?";
+            + "Value Restrictions: -10e9 to 10e9 inclusive";
```

Comment:
Intentional change to -10e9 or?

Reply:
Yes, it is intentional. Currently, `Amount` is also being used for tallying the budget balance, which can be negative, so to be consistent, I have made `Amount` accept values from -10e9 to 0 as well. As for the other uses of `Amount` that does not allow negative values, I have updated the various commands (`AddCommand`, `EditCommand`, `TopupCommand` and `ReduceCommand`) to do the validation instead.